newsfragments/760.feature.rst

```diff
@@ -0,0 +1,3 @@
+We added a new ``is_readable`` method to the ``trio.socket.SocketType``
+object that allows you to check whether a socket is ready to be read
+or not.
```

Comment:
The double-backticks are raw code; since we're referring to objects that have entries in the documentation, we can use single-backticks to link directly:

```
Trio sockets have a new method `~trio.socket.SocketType.is_readable`, that allows you
to check whether a socket is readable. This is useful for HTTP/1.1 clients.
```

(The tilde is a shorthand that means "make a link to `trio.socket.SocketType.is_readable`, but just render it as `is_readable`".)